FirebaseAuth/Tests/Sample/Sample/MainViewController+OAuth.m

```diff
@@ -321,11 +323,57 @@ - (void)signInYahooHeadfulLite {
 - (ASAuthorizationAppleIDRequest *)appleIDRequestWithState:(NSString *)state API_AVAILABLE(ios(13.0)) {
   ASAuthorizationAppleIDRequest *request = [[[ASAuthorizationAppleIDProvider alloc] init] createRequest];
   request.requestedScopes = @[ASAuthorizationScopeEmail, ASAuthorizationScopeFullName];
-  request.nonce = @"REPLACE_ME_WITH_YOUR_NONCE";
+  NSString *nonce = [self randomNonce:32];
```

Comment:
Maybe rename it to `rawNonce` and `self.rawNonce`?

Reply:
Done.